### PR TITLE
DAT-2861 add images to parser output on resolving, use same parser in…

### PR DESCRIPTION
**Pull request by idradm**
…stance for all parsing

### CODE REVIEW

File: extensions/wikia/PortableInfobox/services/Parser/MediaWikiParserService.php [hunk 1]
```diff
@@ -47,10 +46,21 @@ public function parseRecursive( $wikitext ) {
...
 	}
 
+	/**
+	 * Add image to parser output for later usage
+	 * @param string $title
+	 */
+	public function addImage( $title ) {
+		$file = wfFindFile( $title );
+		if ( $file ) {
+			$this->parser->getOutput()->addImage( $file->getName(), $file->getTimestamp(), $file->getSha1() );
```
Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/services/Parser/MediaWikiParserService.php]: Let's handle redlinks (non-existent) files too per offline discussion

**Comment by SebastianMarzjan** [comment]
LGTM

**Comment by idradm** [comment]
DAT-2861 #to-qa

**Comment by idradm** [comment]
DAT-2861 #to-qa LG

**Comment by SebastianMarzjan** [comment]
Still LGTM :)

**Pull request merged**